### PR TITLE
[Devops] added terraform: api-gw stage provision, ami index,transfer …

### PR DESCRIPTION
**Pull request by DaveSys911**
added screenshots and readme

### CODE REVIEW

File: aws/README.md [hunk 1]
```diff
@@ -33,6 +33,17 @@ Terraform init
 Terraform plan -out  <plan_name>
 Terraform apply ./<plan_name>
 ```
+### Setup authorizer on method request:  
+Navigate via aws console to API Gateway Service.
```
Comment (shahar-davidson) [inline aws/README.md:37]:
```suggestion
Navigate via AWS console to the API Gateway Service.   
```

File: aws/Terraform/variables.tf [hunk 2]
```diff
@@ -31,12 +31,15 @@ variable "AMI" {
     
     default = {
         eu-west-2 = "ami-0b6b51e397faf2316"
+        eu-west-3 = "ami-0d9a499b43edd7ae0"
+        us-east-1 = "ami-038f1ca1bd58a5790"
+        us-east-2 = "ami-07a0844029df33d7d"
     }
 }
 
 variable "avail_zn" {
```
Comment (shahar-davidson) [inline aws/Terraform/variables.tf:40]: is this variable being used?

File: aws/Terraform/variables.tf [hunk 3]
```diff
@@ -2,7 +2,7 @@ variable "AWS_REGION" {
  
     default = "eu-west-2"
 }
-
+/*
```
Comment (shahar-davidson) [inline aws/Terraform/variables.tf:5]: why have these been commented out? are they no longer in use?

File: aws/README.md [hunk 4]
```diff
@@ -33,6 +33,17 @@ Terraform init
 Terraform plan -out  <plan_name>
 Terraform apply ./<plan_name>
 ```
+### Setup authorizer on method request:  
+Navigate via aws console to API Gateway Service.   
+Click on "Method Request" (see attached screenshot)   
+![Alt text](./method-request.png?raw=true "Method")
```
Comment (shahar-davidson) [inline aws/README.md:39]: These image files are missing - are you sure you have added them?

**Review by shahar-davidson — CHANGES_REQUESTED** [review]

File: aws/Terraform/variables.tf (hunk 2 above)
Comment (DaveSys911) [reply aws/Terraform/variables.tf:40]: Yes, i just added the relevant amis from others zones, for future usage.

File: aws/README.md (hunk 4 above)
Comment (DaveSys911) [reply aws/README.md:39]: fixed

**Review by shahar-davidson — APPROVED** [review]

**Pull request merged**